Dockerfile

```diff
@@ -110,21 +110,48 @@ RUN file -b   /bin/invoicer
 RUN du        /bin/invoicer
 
 
+# This stage is used to generate /etc/{group,passwd,shadow} files & avoid RUN-ing commands in the `final` layer,
+#   which would break cross-compiled images.
+FROM alpine:3.11 AS perms
+
+# The most generic linux-native name for the user I could quickly come up with 🤷🏻‍♂️
+ARG USER=tux
```

Comment:
Why not a more relevant name to the image like `invoicer`?

Reply:
Yeah, that was my first idea too.

The only concern was that perhaps where will be too much `invoicer` everywhere, but I guess it might be better than a seemingly random name 🤔.

Reply:
Changed